src/queries.ts

```diff
@@ -9,18 +9,26 @@ function querystr_to_array(querystr: string): string[] {
     .map(l => l + ';');
 }
 
-interface QueryParams {
-  bid_window: string;
-  bid_afk: string;
-  bid_browsers?: string[];
+interface BaseQueryParams {
   filter_afk: boolean;
   include_audible?: boolean;
   classes: Record<string, any>;
 }
 
+interface DesktopQueryParams extends BaseQueryParams {
```

Comment:
Instead of having one for desktop and one for android, would it be possible to just have one but force "showAfk" to always be false when running on android so the afk_bucket is ignored?

Reply:
There would still be separate queries since the events have different keys, so I don't think that'd make sense.

Reply:
Ah, didn't know